stock_by_warehouse/static/src/scss/main.scss

```diff
@@ -1,3 +1,9 @@
 .js_product_warehouse, .js_available_info{
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+@import url('enterprise/web_enterprise/static/src/scss/fields.scss');
```

Comment:
The problem was with our widget, there wasn't working the overflow and hasn't a height limit, so we decided to add the import and fix it in case that module is needed.